setup.py

```diff
@@ -26,9 +26,10 @@
       classifiers=classifiers,
       install_requires=[
           'biopython>=1.70',
-          'pandas>=0.22.0',
+          'pandas>=0.23.0',
```

Comment:
Updating pandas and added `dataframe.append(..., sort=True)` to avoid this warning:

```
FutureWarning: Sorting because non-concatenation axis is not aligned. A future version
of pandas will change to not sort by default.
To accept the future behavior, pass 'sort=True'.
To retain the current behavior and silence the warning, pass sort=False
```